autorest/client.go

```diff
@@ -43,6 +28,12 @@ const (
 `
 )
 
+// Response serves as the base for all responses from generated clients. It provides access to the
+// last http.Response.
+type Response struct {
```

Comment:
@ahmetalpbalkan At the time (months ago) it seemed we would need it. Now, not so much. Let's chat about the impact to the SDK.